agents/scsi/fence_scsi.py

```diff
@@ -493,7 +493,11 @@ def main():
 devices must support SCSI-3 persistent reservations (SPC-3 or greater) as \
 well as the \"preempt-and-abort\" subcommand.\nThe fence_scsi agent works by \
 having each node in the cluster register a unique key with the SCSI \
-device(s). Once registered, a single node will become the reservation holder \
+device(s). Reservation key is generated from \"node id\" (default) or from \
+\"node name hash\" (recommended) by adjusting \"key_value\" option. \
```

Comment:
I would say (RECOMMENDED) here to make sure people notice it.

Reply:
commit 6a73919ab70d76fcf4ce19b4fd00e182e41f33b5